contributors/mattercon/mattercon2021.md

```diff
@@ -0,0 +1,31 @@
+
+# MatterCon 2021
+
+MatterCon 20201 will be Mattermost's first VR conference. Using Oculus Quest 2 devices, we'll be meeting up virtually in June 2021. [Join the MatterCon2021 channel on Community](https://community.mattermost.com/core/channels/mattercon-2021). Use the hashtag #mattercon2021.
+
+## When will I receive my Oculus Quest 2?
+
+Some of you may have already received your Oculus Quest 2 headsets and some may still have to wait a bit but everyone should have their Oculus Quest 2 headsets in a couple of weeks to the address and phone number on your BambooHR profile. Please make sure your BambooHR profile is up-to-date!
+
+## Did you receive your Oculus Quest 2?
+
+1. Log in using your Facebook account. If you don't have one or don't want to use your personal Facebook account, feel free to create a Facebook account using your @mattermost.com email.
+2. Download the Oculus app on your phone/mobile device.
+3. A couple of good first free apps to try are called First Steps and Beat Saber Demo.
+4. Download and create an account on the Spatial app on the same phone/mobile device.
+5. Create a realistic avatar on the Spatial app.
+6. Visit [Mattermost VR Gallery](https://app.spatial.io/rooms/609458b76c51d50edeac2669?share=7621477051219851177).
+7. [Pair](https://support.spatial.io/hc/en-us/articles/360043402231-Pair-Your-AR-VR-Headset-to-Spatial) Oculus Quest 2 headset.
+8. Sneak peek at MatterCon 2021 and say hi to the Focalbirds in the [Mattermost VR Gallery](https://app.spatial.io/room/609458b76c51d50edeac2669?share=7621477051219851177).
+10. Help decorate and/or add to the [Mattermost VR Gallery](https://app.spatial.io/rooms/609458b76c51d50edeac2669?share=7621477051219851177).
+
+## Create your own team room (think VR booth at a virtual conference) on Spatial!! Sign up on Focalboard
+
+1. Make your team room by creating a **+NEW SPACE** on your Spatial app with your phone or browser.
+2. Click on the up arrow box on the RHS and change the options to share with **Anyone with a link**.
+3. Go to [Team Rooms Focalboard](https://focalboard-community.octo.mattermost.com/workspace/7kxjjebrnffxdps5bsr8brn9so?id=5601f94e-a5c6-4972-85b4-b815b6c0882f&v=22d9534c-676d-4ea6-8e07-82a76324af95).
```

Comment:
This Focalboard links to an error for me (I tried to log in again)

![image](https://user-images.githubusercontent.com/13119842/117896296-40f79680-b28e-11eb-8bc7-a9f7e11b810e.png)

Reply:
I've had this problem before actually and basically refreshed and tried again - not the ideal solution! @hannaparks, @chenilim - any idea how to resolve this?

Reply:
This either means:
1. You haven't joined the corresponding channel, [MatterCon 2021](https://community.mattermost.com/core/channels/mattercon-2021) in this case
  a. Might be better to just ask folks to join that channel, then click on the Focalboard button in the header
2. Or your (OAuth) session token has expired. It expires every 7 days. To fix, close the tab and follow the steps in 1 above.
  a. There are additional troubleshooting steps in the [Mattermost-Focalboard announcement](https://github.com/mattermost/focalboard/discussions/349). Please LMK if they don't resolve the problem for you. Thanks!

Reply:
Thanks, @chenilim! I've added those steps to the doc. :)